Coinbase Pro.lua

```diff
@@ -119,14 +176,26 @@ function queryCoinbaseProApi(endpoint)
         headers["CB-ACCESS-SIGN"] = MM.base64(apiSign)
         headers["CB-ACCESS-PASSPHRASE"] = apiPassphrase
 
-        local content = Connection():request("GET", url .. path, nil, nil, headers)
-        return JSON(content):dictionary()
+        content, charset, mimeType, filename, rem_headers = Connection():request("GET", url .. path, nil, nil, headers)
+        return JSON(content):dictionary(), rem_headers
 end
 
-function queryExchangeRates(currency)
-        local url = string.format("https://api.pro.coinbase.com/products/%s-%s/ticker", currency, nativeCurrency)
-        local content = Connection():request("GET", url)
-        return JSON(content):dictionary()
+function queryExchangeRates(product_id, products)
+        for _, data in pairs(products) do
```

Comment:
i think its better to have a function to check if products contains product and then to the request instead of loop within this method directly

Reply:
has_value could be used for that?